python no.

```diff
@@ -0,0 +1,21 @@
+# Python program to find the largest number among the three input numbers
+
+# change the values of num1, num2 and num3
+# for a different result
+num1 = 10
+num2 = 14
+num3 = 12
+
+# uncomment following lines to take three numbers from user
+#num1 = float(input("Enter first number: "))
+#num2 = float(input("Enter second number: "))
+#num3 = float(input("Enter third number: "))
+
+if (num1 >= num2) and (num1 >= num3):
+   largest = num1
+elif (num2 >= num1) and (num2 >= num3):
+   largest = num2
+else:
+   largest = num3
```

Comment:
just use the python's max function for a better and more optimized code

Reply:
print(max(a,b,c))

Reply:
In case you are taking input from user:
Let x,y,z be 3 numbers among which you need to find greater one,
Program:
 
x=print(input(value of x))
y=print(input(value of y))
z=print(input(value of z))
print("the highest among x,y,z is",max(x,y,z))